tasks/main.yml

```diff
@@ -0,0 +1,422 @@
+---
+- name: install the EPEL packages required for ODCS backend
+  yum:
+    name: "{{ packages }}"
+    state: present
+    use_backend: yum
+  vars:
+    packages:
+    - python-qpid-proton
+    - python2-koji-cli-plugins
+    - python-psycopg2
+    - python2-celery
+    - mock
+    - dnf
+    - libguestfs-tools-c
+  tags:
+  - odcs
+
+- name: install the internal packages required for ODCS backend
+  yum:
+    name: "{{ packages }}"
+    state: present
+    update_cache: yes
+    use_backend: yum
+  vars:
+    packages:
+    - odcs
+    - jigdo
```

Comment:
This is interesting, just for information do we still require jigdo to produce our artifacts?

Reply:
Its not required anymore.